tools/logger/filter.c

```diff
@@ -0,0 +1,316 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2020 Intel Corporation. All rights reserved.
+ *
+ * Author: Karol Trzcinski	<karolx.trzcinski@linux.intel.com>
+ */
+
+#include <ipc/trace.h>
+#include <smex/ldc.h>
+#include <sof/lib/uuid.h>
+#include <sof/list.h>
+#include <user/trace.h>
+#include <ctype.h>
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "convert.h"
+#include "filter.h"
+
+#define COMPONENTS_SEPARATOR ','
+
+/** map between log level given by user and enum value */
+static const struct {
+	const char name[16];
+	int32_t log_level;
+} log_level_dict[] = {
+	{"verbose",	LOG_LEVEL_VERBOSE},
```

Comment:
vrebose debug ?

Reply:
It's just LOG_LEVEL_* enum representation, in user/trace.h file there is 
`#define LOG_LEVEL_VERBOSE	LOG_LEVEL_DEBUG`